App/Components/Sheets/ModalDrawer.swift

```diff
@@ -0,0 +1,44 @@
+import SwiftUI
+
+/// A view that displays modal content on top of a view that dims the content underneath.
+private struct ModalDrawerContainer<Source: View, Sheet: View>: View {
+    let source: Source
+    let content: Sheet
+    @Binding var isPresented: Bool
+
+    var body: some View {
+        ZStack(alignment: .bottom) {
+            source
+                .zIndex(0)
+                .accessibility(hidden: isPresented)
```

Comment:
Is there a SwiftUI equivalent for [`accessibilityViewIsModal`](https://developer.apple.com/documentation/objectivec/nsobject/1615089-accessibilityviewismodal)?

Reply:
Seems like you need to add `isModal` as an accessibility trait.

Reply:
Actually, it seems like that's not working for some reason? Will leave it like this.